src/main/scala/net/cakesolutions/CakeDockerPlugin.scala

```diff
@@ -24,6 +19,19 @@ object CakeDockerPlugin extends AutoPlugin {
     dockerRepository := None,
     packageName in Docker := name.value,
     maintainer in Docker := "Cake Solutions <devops@cakesolutions.net>",
-    version in Docker := sys.props.get("tag").getOrElse(version.value)
+    version in Docker := sys.props.get("tag").getOrElse(version.value),
+    dockerCommands ++= imageLabels(name.value)
   )
+
+  private def imageLabels(containerName: String): Seq[Cmd] = {
```

Comment:
this doesn't look like it will conflict with our existing project, but could you please explain what it could be used for?